docs/sources/installation/mac.md

```diff
@@ -10,37 +10,61 @@ weight = 500
+   - [Install standalone macOS binaries](#install-standalone-macos-binaries)
+   - [Next steps](#next-steps)
+   - [Upgrade](#upgrade)
 
-**Note on upgrading:** While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "upgrading.md" >}}) for tips and guidance on updating an existing installation.
+**Note on upgrading:** While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Before you perform an upgrade, read [Upgrading Grafana]({{< relref "upgrading.md" >}}) for tips and guidance on updating an existing installation.
 
-## 1. Download and install
+## Install with Homebrew
 
-Before you begin, you must have [Homebrew](http://brew.sh/) installed.
+Use [Homebrew](http://brew.sh/) to install the most recent released version of Grafana using Homebrew package.
+
+1. On the Homberew homepage, search for Grafana. The last stable and released version is listed.  
+1. Open a terminal and enter:
+    ```
+   brew update
+   brew install grafana
+   ```
+
+   The brew page downloads and untars the files into `/usr/local/Cellar/grafana/version`.
+
+1. Start Grafana using the command:
+   ```bash
+   brew services start grafana
+   ```
+
+## Install standalone macOS binaries
+
+To install a nightly build, or to install the latest version of Grafana  without Homebrew, go to the [Grafana download page](https://grafana.com/grafana/download/7.3.0-381ff45epre?platform=mac).
+
+1. Select the Grafana version you want to install. By default, the most recent released version is selected.
+
+   > **Note:** The downloads page lists only finished releases. If you want to install a beta version, click [Nightly ] **Nightly Builds** and then select a version.
 
-1. On the [Grafana download page](https://grafana.com/grafana/download?platform=mac), select the Grafana version you want to install. 
-   * The most recent Grafana version is selected by default.
-   * The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
 2. Select an **Edition**.
    * **Open Source** - Functionally identical to the enterprise version, but you will need to download the enterprise version if you want enterprise features.
    * **Enterprise** - Recommended download. Functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
 3. Click **Mac**.
-4. Copy and paste the code from the installation page into your command line and run. It follows the pattern shown below.
-
+4. Open a terminal and download the binary using the cURL command. The following example shows Grafana 7.1.5 version:
    ```bash
-   brew update
-   brew install grafana
-   ```
+   curl -O https://dl.grafana.com/oss/release/grafana-7.1.5.darwin-amd64.tar.gz
```

Comment:
These instructions do not match what is going on in the official download page: https://grafana.com/grafana/download?platform=mac

I do not want to repeat the downloads page. That's inefficient and annoying for us to maintain. 

Also, we don't want to diverge too much from the official downloads page. If it is wrong, then we need to talk to the website team about fixing it.

Reply:
The download page should not show the brew steps.  That is install information, and moreover, the user can perform those steps from the Homebrew and not the Grafana downloads page.

The macOS section on the download page should be updated.

Reply:
@DanCech, let's chat about this on Monday.